features/step_definitions/page_steps.rb

```diff
@@ -111,21 +111,22 @@
   click_button 'Create Page'
 end
 
-When /^I change the template to "([^"]*)"/ do |template|
+When(/^I change the template to "([^"]*)"/) do |template|
   select template, :from => 'Template'
   click_button 'Update Page'
 end
 
-When /^I change the seo info of the page/ do
+When (/^I change the seo info of the page/) do
   fill_in 'Keywords', :with => 'some_keyword, someother_keyword'
   fill_in 'Description', :with => 'This is a description of the page'
   fill_in 'Browser title', :with => 'This is a new title for the browser'
   click_button 'Update Page'
 end
 
-When /^I change the slug to "([^"]*)"/ do |path|
+When(/^I change the slug to "([^"]*)"/) do |path|
   fill_in 'Slug', :with => path
   click_button 'Update Page'
+  expect(page).to_not have_content('Update Page')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.